cfbs.json

```
@@ -139,6 +139,52 @@
       "subdirectory": "management/every-minute",
       "steps": ["json def.json def.json"]
     },
+    "file-permissions": {
+      "description": "Manages file permissions",
+      "tags": ["management"],
+      "repo": "https://github.com/nickanderson/cfengine-file-permissions",
+      "by": "https://github.com/nickanderson",
+      "version": "0.0.1",
+      "commit": "a678160011b834557f1ab42a081ea84adf8622df",
+      "steps": [
+        "copy file-permissions.cf services/file-permissions.cf",
+        "input file-permissions/input.json def.json",
+        "bundles nickanderson:file_permissions",
```

I don't have a definitive suggestion on what the namespace should be, but perhaps just the canonified module name is okay?
```suggestion
        "bundles file_permissions:file_permissions",
```

---

moving to file_permissions namespce makes me want to rename the bundle to unix or something.